advBatchControl.go

```diff
@@ -60,7 +58,8 @@ func (bc *ADVBatchControl) Parse(record string) {
 
 	// 1-1 Always "8"
 	bc.recordType = "8"
-	// 2-4 This is the same as the "Service code" field in previous Batch Header Record
+	// 2-4 This is the same as the "Service code" field in previous Batch Header Record -
+	// AutomatedAccountingAdvices: 280
```

Comment:
Does this comment need to be here? Not sure what it tells me.

Reply:
Ok I'll remove.  No problem!